### DISCUSSION TITLE
CSS file

### DISCUSSION THREAD

**Discussion by aerdanb** [getgrav/grav#3329] [Help]
Hi! Please,Can someone help me?
On fresh GRAV installation (v1.7.12) updated to (v1.7.13) I installed your thema but it doesn't work properly as you can view in the follow images.
![2021-04-23_1619212623_screenshot](https://user-images.githubusercontent.com/12890691/115930695-35346380-a48a-11eb-8002-3714440c79e7.png)
![2021-04-23_1619212656_screenshot](https://user-images.githubusercontent.com/12890691/115930698-36659080-a48a-11eb-8ea2-73ce2e387219.png)
![2021-04-23_1619212681_screenshot](https://user-images.githubusercontent.com/12890691/115930700-36659080-a48a-11eb-8abd-4fd371926b65.png)
I'm not an expert, not programmer and I cannot understand why the theme doesn't work, please help me.

**Comment by rhukster** [comment]
The theme doesn't seem to be updated for the auto-escaping that was added in Grav 1.7.  You can enable compatibility mode:  https://learn.getgrav.org/17/advanced/grav-development/grav-17-upgrade-guide#twig

**Reply by aerdanb** [reply]
Thank you very much!

I need to set:

- Blueprint Compatibility

- YAML Compatibility

- Twig Compatibility

Grazie mille :)
